.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        type: [alpine, buster-slim]
+        node: [12, 14, 16]
+    env:
+      AZURE_CONTAINER_REGISTRY_URL: hmctspublic.azurecr.io
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Login to Azure Container Repository
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.AZURE_CONTAINER_REGISTRY_URL }}
+          username: ${{ secrets.AZURE_CLIENT_ID }}
+          password: ${{ secrets.AZURE_CLIENT_SECRET }}
+      - name: Build - Node ${{ matrix.node }} ${{ matrix.type }}
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./${{ matrix.type }}
+          platforms: linux/amd64,linux/arm64
+          tags: ${{ env.AZURE_CONTAINER_REGISTRY_URL }}/base/node:${{ matrix.node }}-${{ matrix.type }}
```

hmm I think this is posting PR images as production images automatically.

likely not what we are after? should they go into a different repo / tag?

this is probably a simple fix it doesn't care about the PR number so we don't have to bother cleaning them up it'll just have the latest one, it's a very low traffic repo
```suggestion
          tags: ${{ env.AZURE_CONTAINER_REGISTRY_URL }}/base/node:pr-${{ matrix.node }}-${{ matrix.type }}
```